ktor-server/ktor-server-plugins/ktor-server-double-receive/jvm/test/DoubleReceiveTestJvm.kt

```diff
@@ -13,7 +13,7 @@ class DoubleReceiveTestJvm {
 
     @Test
     fun testFileCache() = runBlocking {
-        val content = ByteArray(16 * 1024 * 1024) { it.toByte() }
```

Comment:
Could you tell me why 16 mb is large?

Reply:
It was burning 500mb in the profiler, which was partly caused by the assert copying all the bytes into objects in the list, but after improving that it was still hitting the cap on TC.  I'll continue with the investigation here, but for now I just want to get the CI working 😅